testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -557,6 +562,15 @@ def test_Cluster_add_metadata(self, cluster):
         cluster.add_metadata('test', metadata)
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
+  
+    def test_empty_Cluster(self, empty):
+        assert empty.size == 0 and np.size(empty.elements) == 0 \
+        and empty.centroid is None and bool(empty.metadata) == False
```

Comment:
It might make sense to break this down into separate assertions on different lines to keep it readable and to keep the feedback on failure a bit more manageable?

I wonder if we need a new fixture for a single usage of `encore.Cluster()`?

Reply:
I should note, you may just be copying the style used throughout this file with super-long assertion lines, but it likely needs updating/modernizing in any case (in a separate PR).

Reply:
Sorry @tylerjereddy must have reviewed at exactly the same time.

Reply:
to make sure that I get your thoughts, 
1- I should delete the "empty" fixture cause it's sort of redundant 
2- I should break the assert line into 4 separate ones
3- make a new pull request after committing those changes

Reply:
You can keep the empty one, just give it a more instructive name. 

Break the asserts into 4 in the same test

Don't make a new PR, just update by pushing new commits to this PR. :)

Reply:
done ✔️